### PR TITLE
Fix ratio for Zhuyin ruby

### PR DESCRIPTION
**Pull request by ethantw**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/clreq/pull/203.html" title="Last updated on Mar 13, 2019, 1:13 AM UTC (4a3ba84)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/clreq/203/9de8be3...4a3ba84.html" title="Last updated on Mar 13, 2019, 1:13 AM UTC (4a3ba84)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2277,42 +2277,42 @@ <h5>
...
+              <p data-lang="zh-hans">所用字体中，平上去入声（ˊˇˋ˪˫ㆴㆵㆶㆷ）及轻声（˙）等调号皆应与汉字尺寸一致，方适用以上比例。</p>
               <p data-lang="zh-hant">所用字體中，平上去入聲（ˊˇˋ˪˫ㆴㆵㆶㆷ）及輕聲（˙）等調號皆應與漢字尺寸一致，方適用以上比例。</p>
             </div>
             <div class="note">
-              <p data-lang="en">In vertical Zhuyin, the width of the space taken by the neutral tone mark does not change but the height should be 10% of the height of the base character. In horizontal Zhuyin, the height of the space taken by the neutral tone mark does not change but the width should be 10% of the height of  the base character.</p>
-              <p data-lang="zh-hans" class="checkme">直排注音时，轻声符宽度不变，高度为基字的10%；横排注音时，轻声符的高度不变，宽度为基字的 10%。</p>
-              <p data-lang="zh-hant">直排注音時，輕聲符寬度不變，高度為基字的10%；橫排注音時，輕聲符的高度不變，寬度為基字的10%。</p>
+              <p data-lang="en">In vertical Zhuyin, the width of the space taken by the neutral tone mark does not change but the height to that of base characters ratio should be 1:15. In horizontal Zhuyin, the height of the space taken by the neutral tone mark does not change but the width to that of base characters ratio should be 1:15.</p>
```
Comment (r12a) [inline index.html]: I suggest: "the height to that of base characters ratio should be " -> "the height ratio to the base character should be"

Comment (ethantw) [reply index.html]: @r12a Thanks, I made the changes.

File: index.html [hunk 2]
```diff
@@ -2277,42 +2277,42 @@ <h5>
...
+            <p data-lang="zh-hans">当注音标注在基字右侧时，原则上占用文字尺寸一半的空间。该空间包含字音与调号，无调号的注音（如国语阴平调）所占的注文空间与包含调号者一致。</p>
             <p data-lang="zh-hant">當注音標注於基字右側時，原則上佔用文字尺寸一半的空間。該空間包含字音與調號，無調號的注音（如國語陰平調）所佔的注文空間與包含調號者一致。</p>
             <div class="note">
               <p data-lang="en">In vertical writing mode, when Zhuyin annotations are placed to the right side of the base characters (vertical Zhuyin), the line gap of the paragraph, where the Zhuyin annotations should be placed, needs more than 1.5 times that of the base characters. The type area should be designed to allow the proper space for the line gap. If the line gap is more than 1.5 times that of the height of the base character, the line gap should not be affected, whether there are Zhuyin annotations between the lines or not.</p>
-              <p data-lang="zh-hans" class="checkme">文字直排、注音置于基字右侧时，段落通常需要基字尺寸1.5倍以上的行距，注文应标注在此行距间，设计版心时须先取适当行距値。段落行距在基字尺寸1.5倍以上时，行距値不应受注音标注与否的影响。</p>
+              <p data-lang="zh-hans">文字直排、注音置于基字右侧时，段落通常需要基字尺寸1.5倍以上的行距，注文应标注在此行距间，设计版心时须先取适当行距値。段落行距在基字尺寸1.5倍以上时，行距値不应受注音标注与否的影响。</p>
```
Comment (willowcheng) [inline index.html]: Is `値` used in Hans? I've only seen `值` in Simplified Chinese. 🤔

Comment (ethantw) [reply index.html]: @willowcheng 謝謝你的提醒，這是我打字時的個人習慣，現在繁簡版都統一使用「值」了。

**Review by xfq — APPROVED** [review]

**Pull request merged**